openedx/features/job_board/views.py

```diff
@@ -114,9 +117,20 @@ class JobDetailView(DetailView):
     template_name = 'features/job_board/job_detail.html'
 
 
-class JobCreateView(CreateView):
+@method_decorator(ensure_csrf_cookie, name='dispatch')
+class JobCreateView(LoginRequiredMixin, CreateView):
+    """
+    Create job view and submit job posting data
+    """
     form_class = JobCreationForm
     template_name = 'features/job_board/create_job_form.html'
 
+    def get_initial(self, *args, **kwargs):  # pylint: disable=unused-argument, arguments-differ
+        """Pre-fill form with initial data"""
+        initial = super(JobCreateView, self).get_initial(**kwargs)
+        user = self.request.user
+        initial['user'] = user.profile.user
```

Comment:
why are we assigning `user.profile.user` instead of just `user`

Reply:
I have to save user extended profile data, just like we are doing for `idea` feature.
https://github.com/philanthropy-u/edx-platform/blob/c531c6565a92a901ce82dad5d8c9ff329fab9fab/openedx/features/idea/views.py#L70